annotations/views/annotation_views.py

```diff
@@ -43,6 +43,15 @@ def relations(request):
 
     paginator = Paginator(qs, 40)
     page = request.GET.get('page')
+
+
+    createdBy = request.GET.get('createdBy')
+    createdAfter = request.GET.get('createdAfter')
+    createdBefore = request.GET.get('createdBefore')
+    project = request.GET.get('project')
+    terminal_nodes = request.GET.get('terminal_nodes')
+    occursIn = request.GET.get('occursIn')
```

Comment:
this is potentially dangerous, you can't trust that the data send in the GET request is safe. It has not been validated.

Reply:
you should try to use the filter object instead, pass that to the template and the retrieve active filters from there.  This would also make sure the code doesn't have to be changed again if more filter fields are added.